virtual_ecosystem/models/plants/plants_model.py

```diff
@@ -164,9 +173,25 @@ def __init__(
         self.model_constant: PlantsConsts
         """Set of constants for the plants model"""
         self.communities: PlantCommunities
-        """Initialise the plant communities from the data object."""
-        self._canopy_layer_indices: np.ndarray
-        """The indices of the canopy layers within wider vertical profile"""
+        """An instance of PlantCommunities providing dictionary access keyed by cell id
+        to PlantCommunity instances for each cell."""
+        self._canopy_layer_indices: NDArray[np.bool_]
+        """The indices of the canopy layers within wider vertical profile. This is 
+        a shorter reference to self.layer_structure.index_canopy."""
+        self.canopies: dict[int, Canopy]
+        """A dictionary giving the canopy structure of each grid cell."""
+        self.filled_canopy_mask: NDArray[np.bool_]
+        """A boolean array showing which layers contain canopy by cell."""
+        self.stem_gpp: dict[int, NDArray[np.float32]]
+        """A dictionary keyed by cell id giving the stem GPP for each cohort in the cell
+        community"""
+        self.stem_transpiration: dict[int, NDArray[np.float32]]
+        """A dictionary keyed by cell id giving the stem transpiration for each cohort
```

Comment:
Stem transpiration is a bit of an odd word, is this the transpiration per individual? Just for clarification, no need to change

Reply:
Yes - I've changed this to `per_stem_gpp` and `per_stem_transpiration`

Reply:
Aha 😆  I think it is clearer with `per_stem_transpiration` that we don't mean the stem is transpiring, so I think this is OK!